src/components/Tag/TagDetails.jsx

```diff
@@ -219,7 +219,28 @@ function TagDetails() {
           />
         );
       case 'ReferredBy':
-        return <ReferredBy referrers={imageDetailData?.referrers} />;
+        // Create a union of manifest referrers and index referrers
+        // In case of singlearch images the manifest and index referrers are the same
+        // In case of multiarch images the index may have referrers and the manifest may have other referrers
+        // Since we don't have a separate widget for index referrers bs manifest referrers, we need to show all of them here
+
+        let allReferrers = [];
```

Comment:
We can write this logic a bit more cleanly using lodash's uniqBy 
something like
```
const allReferrers = uniqBy([...selectedManifest.referrers, ...imageDetailData.referrers], 'digest')
```